qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -448,8 +504,8 @@ def backends(
         Args:
             name: Backend name to filter by.
             min_num_qubits: Minimum number of qubits the backend has to have.
-            instance: This is only supported for ``ibm_quantum`` runtime and is in the
-                hub/group/project format.
+            instance: In hub/group/project format if on the ``ibm_quantum`` channel.
+                IBM Cloud account crn if on the ``ibm_quantum_platform`` channel.
```

Comment:
Same comment on still documenting "ibm_cloud" if it is still a valid channel, which it is until deprecation (then during the deprecation PR we can mark it as deprecated in the docstrings, but not remove the info until the feature is removed).